src/loader-components/ExampleTemplate.module.scss

```diff
@@ -39,6 +41,7 @@
   .bg-toggle,
   .code-toggle {
     align-items: center;
+    background-color: var(--terra-dev-site-example-loader-code-toggle-background-color, #ddd);
```

Comment:
Things are a little inconsistent in this file, I think the pattern should be:

Prefix:

`--terra-dev-site-example-template`

Example:

`--terra-dev-site-example-template-code-toggle-background-color`